workers.py

```diff
@@ -84,13 +86,15 @@ def ai_service_worker(
         b64_identity: str = None,
         ) -> Thread:
     """Outlier detection."""
+    @REQUEST_TIME.time()
     def worker() -> None:
         thread = current_thread()
         LOGGER.debug('%s: Worker started', thread.name)
 
         try:
             account_id, batch_data = job['account'], job['data']
             rows = batch_data['total']
+            METRICS['data_size'].observe(rows)
```

Comment:
How about showing the `data_size` in bytes rather than rows?
(this has been done in https://github.com/ManageIQ/aiops-data-collector/pull/41)

We could put the number of rows in the log instead.

Reply:
This is number of systems in the data set. This piece of information is very useful. Would like it to be seen on charts, instead from grepping from logs (it's already in logs)

Reply:
Ok, in that case should this be recorded via the data-collector metrics instead?

Reply:
you can choose to do that in data-collector, but this number is even more meaningful in the context of `rad`.